### DISCUSSION TITLE
OTBR spinel drivers to support thread networking for client / controller

### DISCUSSION THREAD

**Discussion by adminy** [matter-js/matter.js#254] [Feature requests (vote via thumbs up)]
with the help of https://github.com/openthread/pyspinel we can communicate with a thread capable processor the same way we can do for bluetooth or wifi. This is particularly great as having a congested wifi / lan network is usually a bad idea and if we can communicate through other channels like thread we have yet another way to interact with devices and create devices. Another benefit of thread is its sometimes longer distances ranges since its a low data rate protocol.

Support thread for:
- [ ] controller device
- [ ] client / bridged device

**Comment by Apollon77** [comment]
Without having looked to deep the "Todo" here would be first to have a spinel implementation in JS/TS first (in separate package) and then integrate this.

Additionally based on this we then need a "Thread network commissioning" implementations with all command handlers for scanning, setting and connecting to a Thread network during the commissioning process.
